clients/devops_challenge_component/src/devops_challenge/network/mutations/triggerAssessment.ts

```diff
@@ -0,0 +1,23 @@
+import { devOpsChallengeAxiosInstance } from '../devOpsChallengeServerConfig'
+
+export const triggerAssessment = async (
+  gitHubHandle: string,
+  coursePhaseID: string,
+): Promise<string> => {
+  try {
+    const response = await devOpsChallengeAxiosInstance.post(
```

Comment:
Wenn du ganz clean sein möchtest, kannst du hier beim post noch den typ hinzufügen, wie du es in der anderen Mutation gemacht hast
```suggestion
    const response = await devOpsChallengeAxiosInstance.post<string>(
```